src/IncrementalChecker.ts

```diff
@@ -128,9 +133,29 @@ class IncrementalChecker {
     return endsWith(filePath, '.d.ts') || linterExclusions.some(matcher => matcher.match(filePath));
   }
 
+  /**
+   * Check for vue-loader in user's project node_modules.
+   */
+  hasVueLoader(): boolean {
+    if (this.vueLoaderResolved !== undefined) return this.vueLoaderResolved;
```

Comment:
Please add curly braces to follow code style in the project :) Would be good to add prettier to this project in a feature